api/src/main/java/org/hawkular/inventory/api/Relationships.java

```diff
@@ -54,6 +54,30 @@ private Relationships() {
         owns
     }
 
+
+    /**
+     * The list of possible relationship (aka edges) direction. Relationships are not bidirectional.
+     */
+    public enum Direction {
+        /**
+         * Relative to the current position in the Hawkular pipeline, this value expresses such relationships
```

Comment:
Hawkular pipeline is an impl detail in the tinkerpop impl. I'd not mention it here in the API. Maybe use "inventory traversal" instead?